src/components/RecipeDetailMain.js

```diff
@@ -3,8 +3,8 @@ import PropTypes from 'prop-types';
 import { withRouter } from 'react-router-dom';
 
 class RecipeDetailMain extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
```

Comment:
O que acha de remover essa `props` já que não esta sendo usada?
```suggestion
  constructor() {
    super();
```